README.md

```diff
@@ -78,25 +76,30 @@ if (obj1.sketchObject == obj2.sketchObject) {
 Here's a very simple example script:
 
 ```javascript
-var sketch = context.api()
+var api = SketchAPI
```

Comment:
If we're going to have a global, I'd be inclined to just call it `sketch`. Whatever we pick is bound to clash with someone somewhere, so we might as well choose the thing that will look cleanest when people use it directly.

I think `sketch.version` looks cleaner than `SketchAPI.version`, and for the purposes of scripts that use it, the API _is_ Sketch.

Reply:
:+1: